Cargo.toml

```diff
@@ -69,10 +69,16 @@ shellexpand = { version = "3.1.0", optional = true }
 
 [features]
 default = []
-all = ["electrum", "esplora", "fs", "cli", "clap", "log"]
-cli = ["base64", "env_logger", "clap", "shellexpand", "fs", "serde", "electrum", "esplora", "log"]
+all = ["electrum", "mempool", "fs", "cli", "clap", "log"]
+cli = ["base64", "env_logger", "clap", "shellexpand", "fs", "serde", "electrum", "mempool", "log"]
```

Comment:
For now this is correct because `mempool` is just an alias for `esplora`, but one day the `mempool` feature could change (it could drop `esplora`) and by mistake we might forget to re-include `esplora` to `all`/`cli` features. Therefore I suggest adding `mempool` to `all` and `cli` instead of replacing `esplora` with `mempool`